pfs_middleware/pfs_middleware/utils.py

```diff
@@ -133,7 +135,7 @@ def call(self, rpc_request, timeout):
                             % (err, line))
 
                 errstr = response.get("error")
-                if errstr:
+                if errstr and raise_on_rpc_error:
```

Comment:
Can we "adorn" this error/exception report with, say, a "PROXYFS" tag so that it is easy to search for in the log? As written, each individual error would only share "Error in" in common... I dare say there may be other lines with those two words in them, no?

Reply:
Are you thinking of just the new `PROXYFS`-method requests, or all `RpcError`s generally?

The point of adding the `raise_on_rpc_error` arg is that `PROXYFS` requests won't raise these errors at all -- they'll just return 422 with a JSON body that (should) exactly match the response from proxyfsd.

When the "normal" methods fail in such a way as to trip an `RpcError`, that should get caught and logged around https://github.com/swiftstack/ProxyFS/blob/1.10.0/pfs_middleware/pfs_middleware/middleware.py#L787-L802 where it'll pick up `RPC error:`, which should be rather unique to ProxyFS...

Reply:
I *was* thinking of just the new PROXYFS method... but take your point.

Let me work for it as is... and see how it goes.